locales/en-US/app.ftl

```diff
@@ -214,6 +214,7 @@ IdleSearchField--search-input =
 ## JSTracer is an experimental feature and it's currently disabled. See Bug 1565788.
 
 JsTracerSettings--show-only-self-time = Show only self time
+    .title = Show only the time spent in that call node, ignoring its children.
```

Comment:
I'd suggest
```suggestion
    .title = Show only the time spent in a call node, ignoring its children.
```
Because this is for all displayed call nodes, not a specific one.

Reply:
What do you think of using the same title for "invert call stack"?

Reply:
Yeah, I think that makes sense to use that in the "invert call stack" as well. I feel like it's more than only this explanation. but "invert" probably doesn't require an explanation anyway.